packages/api/src/graphql/comment.ts

```diff
@@ -219,7 +233,10 @@ export const GraphQLPublicComment: GraphQLObjectType<
 
     rejectionReason: {type: GraphQLString},
 
-    modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)}
+    modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
+    rating: {
+      type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLRating)))
```

Comment:
The mean should also come on level RatingSystemAnswer.

Reply:
Why?